include/mbgl/style/expression/format_expression.hpp

```diff
@@ -25,22 +25,22 @@ struct FormatExpressionSection {
 
 class FormatExpression final : public Expression {
 public:
-    explicit FormatExpression(std::vector<FormatExpressionSection> sections);
+    explicit FormatExpression(std::vector<FormatExpressionSection> sections) noexcept;
```

Comment:
```suggestion
    explicit FormatExpression(std::vector<FormatExpressionSection> sections);
```

Not `noexcept`, the `Expression` constructor is not.